### PR TITLE
fixes #18980 - ssh keys new page has breadcrumbs

### PR DESCRIPTION
**Pull request by timogoebel**
image

### CODE REVIEW

**Comment by ohadlevy** [comment]
:+1:  overall, I would consider dropping the home link?
/cc @rohoover ?

**Comment by dLobatog** [comment]
👍 too after @rohoover's comment

**Comment by Rohoover** [comment]
Yes, drop home, thanks!

**Comment by timogoebel** [comment]
Dropped the home link.

File: app/views/ssh_keys/new.html.erb [hunk 1]
```diff
@@ -1,3 +1,9 @@
 <% title _('Create SSH Key for %s') % @user.name %>
 
+<ol class="breadcrumb">
+  <li class="breadcrumb-item"><%= link_to_if_authorized(_("Users"), hash_for_users_path) %></li>
+  <li class="breadcrumb-item"><%= link_to_if_authorized(@user.name, hash_for_edit_user_path(@user)) %></li>
+  <li class="breadcrumb-item active"><%= _('Create SSH Key') %></li>
```
Comment (ohadlevy) [inline app/views/ssh_keys/new.html.erb:6]: is there an edit scenario as well?

Comment (timogoebel) [reply app/views/ssh_keys/new.html.erb:6]: @ohadlevy: No, you can not edit SSH keys right now. Just delete them or create new ones.

**Comment by ohadlevy** [comment]
thanks @timogoebel

**Pull request merged**